### PR TITLE
Implement POST Primary Validator Updated for `ConfirmationValidator`

### LINKED ISSUE #30: [Task] Implement POST Primary Validator Updated for `ConfirmationValidator`
# Task

Contributors are able to complete this task and earn thenewboston coins.  Check out the labels to learn how much you can earn for contributing by completing this task.  Please make sure to be honest if you wish to contribute by saying you can't finish this and we can just un-assign you with no harm done!  There is no point in delaying tasks from being completed for miscommunication!

## Overview

Implement the POST `/primary_validator_updated` endpoint as a `ConfirmationValidator` method. This method should take in the `ipAddress`, `protocol`, `port`, and `serverAccount` of the bank server that is receiving the update ping. Refer the [docs on the `/primary_validator_updated` endpoint](https://thenewboston.com/confirmation-validator-api/primary-validator-updated) and the [`Bank.updateAccount` method](https://github.com/thenewboston-developers/thenewboston-js/blob/master/src/bank.ts#L15-L23) (of course use `postData` instead of `patchData`).

## Behavior

The `ConfirmationValidator` method that you are creating should have the following shape:

```ts
class ConfirmationValidator {
  /**
   * Sends a notification to the bank that a primary validator has left the network.
   * @param ipAddress the ip address of the primary validator that is leaving
   * @param port the port that the primary validator is on
   * @param protocol the protocol of the primary validator
   * @param serverAccount the account that the current `ConfirmationValidator` is connected to
   */
  async sendPrimaryValidatorUpdatedPing(ipAddress: string, port: string, protocol: string, serverAccount: Account): Promise<any>
}
```

Please remember to put the JSDoc above the method in your actual code.

### PR DESCRIPTION
**Pull request by AbhayAysola**
closes #30 
Account Number: 8928526805de48e4bf2ed2a9b4b839e6b2603018ecdfbf4cffdb2065e01a3ed1

### CODE REVIEW

File: src/confirmation-validator.ts [hunk 1]
```diff
@@ -11,7 +12,24 @@ export class ConfirmationValidator extends Validator {
...
+   */
+  async sendPrimaryValidatorUpdatedPing(
+    ipAddress: string,
+    port: string,
+    protocol: string,
+    serverAccount: Account
+  ): Promise<any> {
+    return await this.postData(
+      "/primary_validator_updated",
+      serverAccount.createSignedMessage({ ip_address: ipAddress, port: port, protocol })
```
Comment (sno2) [inline src/confirmation-validator.ts:30]: Please use the shorthand for `port`.

File: src/confirmation-validator.ts [hunk 2]
```diff
@@ -11,7 +12,24 @@ export class ConfirmationValidator extends Validator {
...
+   * @param port the port that the primary validator is on
+   * @param protocol the protocol of the primary validator
+   * @param serverAccount the account that the current `ConfirmationValidator` is connected to
+   */
+  async sendPrimaryValidatorUpdatedPing(
+    ipAddress: string,
+    port: string,
+    protocol: string,
+    serverAccount: Account
+  ): Promise<any> {
```
Comment (sno2) [inline src/confirmation-validator.ts:27]: Please don't include the return type as that is implicit.  Sorry that my prompt may have inclined you to put it there.

**Review by sno2 — CHANGES_REQUESTED** [review]
Thanks, please read my requests for changes.

File: src/confirmation-validator.ts (hunk 2 above)
Comment (AbhayAysola) [reply src/confirmation-validator.ts]: No problem. Fixed that

File: src/confirmation-validator.ts (hunk 1 above)
Comment (AbhayAysola) [reply src/confirmation-validator.ts]: Sure

**Comment by sno2** [comment]
LGTM, thanks!

**Pull request merged**